tests/php/src/DevTools/CallbackReflectionTest.php

```diff
@@ -39,7 +39,10 @@ public function setUp() {
 
 		$this->register_core_themes();
 
-		$this->callback_reflection = Services::get( 'injector' )->make( CallbackReflection::class );
+		$plugin_registry = new PluginRegistry();
+		$file_reflection = new FileReflection( $plugin_registry );
+		$file_reflection->register();
```

Comment:
Not necessarily something you should do here, but wanted to clarify nevertheless:

The `FileReflection` service is a "shared" instance: https://github.com/ampproject/amp-wp/pull/5289/commits/886022d51bbcfe463c507e33aa8981e81e5c674d#diff-e12f38193bbbb46cce767c749eaebf1bR146-R153

This means that both the injector and the service locator will always return the same object instance. So you could for example use the injector to make `CallbackReflection` and then use the service locator to call the `register()` method on the `FileReflection` and it would hit the one that was already injected into the `CallbackReflection`.

Reply:
No idea where this link is actually going to... 😕 

Here's what I wanted to show:
![Image 2020-09-13 at 10 16 12 AM](https://user-images.githubusercontent.com/83631/93013615-345d5a00-f5aa-11ea-953d-4aba7dfd0bd3.png)